editor.planx.uk/src/@planx/components/Checklist/Editor/Options.tsx

```diff
@@ -38,15 +38,15 @@ export const Options: React.FC<{ formik: FormikHookReturn }> = ({ formik }) => {
               formik.setFieldValue("options", newCombinedOptions);
             }}
             newValueLabel="add new option"
-            newValue={() =>
-              ({
-                data: {
-                  text: "",
-                  description: "",
-                  val: "",
-                },
-              }) as Option
```

Comment:
This casting as masking the problem - `flags` was toggling between `undefined` and an actual value on change.